bokehjs/src/coffee/models/tools/gestures/wheel_zoom_tool.coffee

```diff
@@ -51,13 +46,7 @@ class WheelZoomToolView extends GestureTool.View
 
     factor  = @model.speed * delta
 
-    zoom_info = scale_range({
-      frame: frame
-      factor: factor
-      center: [vx, vy]
-      v_axis_only: v_axis_only
-      h_axis_only: h_axis_only
-    })
+    zoom_info = scale_range(frame, factor, h_axis=h_axis, v_axis=v_axis, {x: vx, y: vy})
```

Comment:
@canavandl, by name arguments don't work in coffeescript. This will be translated directly and it will result in whatever else behavior than you expect. If you want "by name arguments", you should have left the original API with a plain object and its destruction.